env.template

```diff
@@ -1,25 +1,62 @@
 # Copy this file to ".env.dev" and ".env.docker.dev", and then tweak values for local development
+
+# Port that the application REST endpoints listen on
+API_PORT=3000
+
+# Blockchain node address
 FREQUENCY_URL=ws://0.0.0.0:9944
+
+# Specifies the provider ID
 PROVIDER_ID=1
+
+# Base URL for provider webhook endpoints
 PROVIDER_BASE_URL=https://some-provider/api/v1.0.0
+
+# Number of connection/page to request when requesting provider connections from webhook
 CONNECTIONS_PER_PROVIDER_RESPONSE_PAGE=100
+
+# Redis URL
 REDIS_URL=redis://0.0.0.0:6379
+
+# An optional bearer token authentication to the provider webhook
 PROVIDER_ACCESS_TOKEN=some-token
+
+# How many minutes to delay between successive scans of the chain
+# for new accounts (after end of chain is reached)
 BLOCKCHAIN_SCAN_INTERVAL_MINUTES=1
+
+# Max number of jobs allowed on the 'graphUpdateQueue' before
+# blockchain scan will be paused to allow queue to drain
 QUEUE_HIGH_WATER=1000
+
+# Seed phrase for provider MSA control key
 PROVIDER_ACCOUNT_SEED_PHRASE='come finish flower cinnamon blame year glad tank domain hunt release fatigue'
+
+# Number of failures allowing in the provider webhook before the service is marked down
 WEBHOOK_FAILURE_THRESHOLD=3
-HEALTH_CHECK_SUCCESS_THRESHOLD=10
+
+# Number of seconds between provider webhook retry attempts when failing
 WEBHOOK_RETRY_INTERVAL_SECONDS=10
-HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS=10
+
+# Number of `/health` endpoint failures allowed before marking the provider webhook service down
 HEALTH_CHECK_MAX_RETRIES=4
+
+# Number of seconds to retry provider webhook `/health` endpoint when failing
+HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS=10
+
+# Minimum number of consecutive successful calls to the provider webhook
+# `/health` endpoint before it is marked up again
+HEALTH_CHECK_SUCCESS_THRESHOLD=10
+
+# Maximum amount of provider capacity this app is allowed to use (per epoch)
+#     type: 'percentage' | 'amount'
+#     value: number (may be percentage, ie '80', or absolute amount of capacity)
 CAPACITY_LIMIT='{"type":"percentage", "value":80}'
```

Comment:
I found at least for me, I had to remove the `'` marks:

```suggestion
CAPACITY_LIMIT={"type":"percentage", "value":80}
```

Reply:
Now that I look closer, it was all inside of Docker the issues happened. So it was the `.env.docker.dev` where the issue was occurring.